spec/features/tags/user_sees_tags_spec.rb

```diff
@@ -17,9 +17,22 @@
       job2 = Job.create(title: "Systems Architect", level_of_interest: 74, city: "Miami", company_id: company.id)
       job.jobs_tags.create(tag: tag)
       job2.jobs_tags.create(tag: tag)
-      byebug
+
       visit company_job_path(company, job)
 
       expect(page).to have_content("Computer - 2")
     end
+
+    scenario "sees average salary for tags" do
+      # As a user,
+      company = Company.create(name: "Xcel Energy")
+      tag = Tag.create(title: "Computer")
+      job = Job.create(title: 'Software Engineer', level_of_interest: 100, city: "Denver", salary: 90000, company_id: company.id)
+      job2 = Job.create(title: "Systems Architect", level_of_interest: 74, city: "Miami", salary: 120000, company_id: company.id)
+      job.jobs_tags.create(tag: tag)
+      job2.jobs_tags.create(tag: tag)
+      # When I visit a specific job page,
```

Comment:
Would love to see less reliance on the join table in your tests. Remember, if there is a many to many relationship established, you can call `job.tags.create` and that will create the JobTags record for you.